azure-communication-ui/chat/checkstyle.gradle

```diff
@@ -0,0 +1,15 @@
+apply plugin: 'checkstyle'
+
+checkstyle {
+    description 'Check style for the Java code files'
+    configFile file('../azure-communication-ui/chat/checkstyle/checkstyle.xml')
+}
+
+task checkstyle(type: Checkstyle) {
+    source 'src'
+    include '**/*.java'
+    exclude '**/gen/**'
+    exclude '**/R.java'
+    exclude '**/BuildConfig.java'
+    classpath = files()
```

Comment:
we might want to include .kt too?

Reply:
As I mentioend above, I remember for calling, this checkstyle thing is only for Java code